ratelimiting/ratelimiting_user.c

```diff
@@ -409,9 +382,6 @@ int main(int argc, char **argv)
     while(1)
     {
         sleep(60);
```

Comment:
can't we get rid of the sleep() and use pause() and put the ffflush(info) at top/=?